### PR TITLE
this is my project. it is ready

### PR DESCRIPTION
**Pull request by olegkoropec**

### CODE REVIEW

File: src/main/java/bridge/CapitanBridgeMain.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package bridge;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import static reference.Answers.*;
+
+public class CapitanBridgeMain {
```
Comment (Volodymyr95) [inline src/main/java/bridge/CapitanBridgeMain.java]: CapitanBridgeService

File: src/main/java/bridge/CapitanBridge.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/take-call2")
+public class CapitanBridge extends HttpServlet {
```
Comment (Volodymyr95) [inline src/main/java/bridge/CapitanBridge.java]: CapitanBridgeServlet

File: src/main/java/passenger/PassengerService.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
+package passenger;
+
+import exceptions.InvalidParamException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+public class PassengerService {
```
Comment (Volodymyr95) [inline src/main/java/passenger/PassengerService.java]: why do u need this?

File: src/main/java/startConversation/StartConversationMain.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
+package startConversation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static reference.Answers.APPROVE_PAGE2;
+import static reference.Answers.DISAPPROVE;
+
+public class StartConversationMain {
```
Comment (Volodymyr95) [inline src/main/java/startConversation/StartConversationMain.java]: make more informative name

File: src/main/java/passenger/PassengerService.java (hunk 3 above)
Comment (olegkoropec) [reply src/main/java/passenger/PassengerService.java]: here I check the correctness of the data entry and if something is wrong, then the error code 5 is thrown out with a notification

File: src/main/java/service/PassengerService.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
+package service;
+
+import exception.InvalidParamException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import passenger.Passenger;
+import java.util.regex.Pattern;
+
+public class PassengerService {
```
Comment (Volodymyr95) [inline src/main/java/service/PassengerService.java]: why do u need this here?

File: src/main/java/service/CapitanBridgeService.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static reference.Answer.*;
+import static reference.ToPage.*;
+
+public class CapitanBridgeService {
+    public static final Logger LOGGER = LoggerFactory.getLogger(CapitanBridgeService.class);
+
+    public String whoAreYou(String result) {
```
Comment (Volodymyr95) [inline src/main/java/service/CapitanBridgeService.java]: is it correct name for this class method?

File: src/main/java/service/StartConversationService.java [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static reference.Answer.*;
+import static reference.ToPage.*;
+
+public class StartConversationService {
+    public static final Logger LOGGER = LoggerFactory.getLogger(StartConversationService.class);
+
+    public String toBridge(String result) {
```
Comment (Volodymyr95) [inline src/main/java/service/StartConversationService.java]: use informative methods name

File: src/main/java/web/GoToQues.java [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+import static reference.ToPage.APPROVE_PAGE1;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+@WebServlet("/take-call0")
```
Comment (Volodymyr95) [inline src/main/java/web/GoToQues.java]: not informative url

**Review by Volodymyr95 — APPROVED** [review]

**Pull request merged**